src/base/QXmppBitsOfBinaryDataList.h

```diff
@@ -18,6 +19,8 @@ class QXMPP_EXPORT QXmppBitsOfBinaryDataList : public QVector<QXmppBitsOfBinaryD
     QXmppBitsOfBinaryDataList();
     ~QXmppBitsOfBinaryDataList();
 
+    std::optional<QXmppBitsOfBinaryData> forCidUri(const QString &uri) const;
```

Comment:
Can you add a `QXmppBitsOfBinaryContentId` overload or change the type to `QXmppBitsOfBinaryContentId`?